libnavigation-core/src/main/java/com/mapbox/navigation/core/navigator/TilesetDescriptorFactory.kt

```diff
@@ -39,45 +39,18 @@ class TilesetDescriptorFactory internal constructor(
         )
 
     /**
-     * Creates TilesetDescriptor using the specified dataset and latest locally available version.
-     *
-     * @param tilesDataset string built out of `<account>[.<graph>]` variables.
-     * Account can be `mapbox` for default datasets or your username for other.
-     * Graph can be left blank if you don't target any custom datasets.
-     * If null [RoutingTilesOptions.tilesDataset] will be used.
-     * @param tilesProfile profile of the dataset.
-     * If null [RoutingTilesOptions.tilesProfile] will be used.
+     * Gets TilesetDescriptor which corresponds to current Navigator dataset
+     * and the specified `tilesVersion`.
+     * @param tilesVersion TilesetDescriptor version
      */
-    @JvmOverloads
-    fun buildLatestLocal(
-        tilesDataset: String? = null,
-        tilesProfile: String? = null
-    ): TilesetDescriptor =
-        nativeFactoryWrapper.buildLatestLocal(
-            cache,
-            combineDatasetWithProfile(tilesDataset, tilesProfile)
-        )
+    fun getSpecificVersion(tilesVersion: String): TilesetDescriptor =
+        nativeFactoryWrapper.getSpecificVersion(cache, tilesVersion)
 
     /**
-     * Creates TilesetDescriptor using the specified dataset and latest version retrieved from
-     * the server.
-     *
-     * @param tilesDataset string built out of `<account>[.<graph>]` variables.
-     * Account can be `mapbox` for default datasets or your username for other.
-     * Graph can be left blank if you don't target any custom datasets.
-     * If null [RoutingTilesOptions.tilesDataset] will be used.
-     * @param tilesProfile profile of the dataset.
-     * If null [RoutingTilesOptions.tilesProfile] will be used.
+     * Gets TilesetDescriptor which corresponds to the latest available version of routing tiles.
+     * Intended for using when creating off-line tile packs.
```

Comment:
Is this sentence needed? Is `getSpecificVersion` not recommended to use for some reason?

Reply:
copied from NN docs. Looks like they mean we should use it to avoid mistakes and load the same version as navigator uses (it uses the latest if customer haven't set another)
cc @DmitryAk

Reply:
- `getSpecificVersion(...)` requires to know and specify the exact version. If it is required to get a `TilesetDescriptor` instance corresponding to some specific version - `getSpecificVersion` can be used, no problem. E.g., if it is required to get a `TilesetDescriptor` corresponding to some previously downloaded off-line tile pack with known version - this method can be used.

- `getLatest(...)` doesn't require to specify the version (you don't even have to know it) - it is resolved to the latest available version. Also it is expected that off-line tile packs will be created using this `TilesetDescriptor` (so they will be based on the latest available version)

Reply:
I removed it.